### PR TITLE
OSDOCS-5553: moving content from config to running apps

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5553

Link to docs preview:
https://57368--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-operators.html
https://57368--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-tools.html

QE review:
- Content moves and copy edits only, SME approval

Additional information:
Follows up on https://github.com/openshift/openshift-docs/pull/57349

### CODE REVIEW

File: _topic_maps/_topic_map_ms.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ Name: Running applications
 Dir: microshift_running_apps
 Distros: microshift
 Topics:
- - Name: Running applications
+ - Name: Operators
```
Comment (dhellmann) [inline _topic_maps/_topic_map_ms.yml:141]: This isn't just about operators.

```suggestion
 - Name: Deploying workloads
```

**Review by dhellmann — COMMENTED** [review]
Let's talk about these changes before proceeding.

**Review by kelbrown20 — COMMENTED** [review]
Hey Shauna! Great work! Just some nits and suggestions, lmk if you have any questions or concerns. Thank you!

**Comment by stevsmit** [comment]
@ShaunaDiaz I left a few more nits here for your consideration. Feel free to DM me when/if you decide to address them. None of them are pressing, but should maybe be considered.

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**